drivers/i2s/Kconfig.mcux_flexcomm

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config I2S_MCUX_FLEXCOMM
+	bool "MCUX FLEXCOMM I2S driver"
+	depends on HAS_MCUX_FLEXCOMM
+	select DMA
+	select INIT_AUDIO_PLL
+	help
+	  Enable support for mcux flexcomm i2s driver.
+
+if I2S_MCUX_FLEXCOMM
+
+config I2S_MCUX_FLEXCOMM_RX_BLOCK_COUNT
+	int "RX queue length"
```

Comment:
Does it make sense to have these as devicetree properties?

Reply:
I followed the approach taken by other I2S drivers.